crates/ruff_server/docs/setup/NEOVIM.md

```diff
@@ -85,7 +85,7 @@ require('lspconfig').ruff.setup {
   init_options = {
     settings = {
       logLevel = "debug",
-      logFile = "your/log/file/path/log.txt"
+      logFile = "~/.config/nvim/logs/ruff_logs.txt"
```

Comment:
nit: logs for Neovim are usually stored under `~/.local/state/nvim`, you can find out the path by running `:echo stdpath('log')` in Neovim :)